backends/platform/dc/dc.h

```diff
@@ -60,13 +60,14 @@ class DCCDManager : public DefaultAudioCDManager {
 public:
 	// Poll cdrom status
 	// Returns true if cd audio is playing
-	bool isPlaying() const;
+	bool isPlaying() const override;
```

Comment:
override is C++11, which is not supported by g++ 4.6.3 which I'm using to build the DC binary dists.

Reply:
We have a `#define` for `override` in `common/c++11-compat.h` to provide compatibility when using a compiler that does not support c++11. So this should be fine as long as `common/scummsys.h` is in the include chain (as this is where `common/c++11-compat.h` gets included).

Reply:
Ah, ok.  `backends/platform/dc/dc.h` includes `backends/base-backend.h`, which includes `common/system.h`, which does include `common/scummsys.h`.  So that should be all right then.  I would have preferred if the macro was called `OVERRIDE` or something so you knew it was a macro definition, but at least it should compile then.